src/modules/fancyzones/FancyZonesEditorCommon/FancyZonesEditorCommon.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\..\Version.props" />
+
+  <PropertyGroup>
+    <TargetFramework>net7.0-windows</TargetFramework>
```

Comment:
We're on net8 already on main. This one is lagging behind.

Reply:
Thanks for noticing, fixed!